gef.py

```diff
@@ -10815,39 +10604,39 @@ class GefHelpCommand(gdb.Command):
     _cmdline_ = "gef help"
     _syntax_  = _cmdline_
 
-    def __init__(self, commands, *args, **kwargs):
+    def __init__(self, commands: List[Tuple[str, Any, Any]], *args, **kwargs) -> None:
         super().__init__(self._cmdline_, gdb.COMMAND_SUPPORT, gdb.COMPLETE_NONE, False)
         self.docs = []
         self.generate_help(commands)
         self.refresh()
         return
 
-    def invoke(self, args, from_tty):
+    def invoke(self, args, from_tty) -> None:
         self.dont_repeat()
         gef_print(titlify("GEF - GDB Enhanced Features"))
-        gef_print(self.__doc__)
+        gef_print(self.__doc__ or "")
         return
 
-    def generate_help(self, commands):
+    def generate_help(self, commands: List[Tuple[str, Type, Any]]) -> None:
```

Comment:
```suggestion
    def generate_help(self, commands: List[Tuple[str, Any, Any]]) -> None:
```

To be consistent with `__init__` above and `add_command_to_doc` below